OneSignalSDK/onesignal/core/src/main/java/com/onesignal/common/modeling/SingletonModelStore.kt

```diff
@@ -38,10 +38,12 @@ open class SingletonModelStore<TModel>(
         tag: String,
     ) {
         synchronized(replaceLock) {
+            val oldModel = this.store.create() ?: throw Exception("Unable to initialize model from store $store")
+            oldModel.initializeFromModel(singletonId, this.model)
```

Comment:
Hmm, this is a bit convoluted workaround and do prefer to check if there is even an old model